challenge.rb

```diff
@@ -22,7 +22,7 @@ def is_of_age? age, minimum_age
 # Remember: `p`, `print`, and `puts` don't return their arguments.
 
 def okay_to_drink? age, drinking_age
-  # Replace this comment with your code.
+  age > drinking_age
```

Comment:
You were supposed to solve this by invoking `is_of_age?`; also, your code doesn't return the string that was asked for.